src/patternfly/layouts/Bullseye/docs/code.md

```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-Use a bullseye layout when there is a single child element, and that child is centered both vertically and horizontally in the parent.
+The Bullseye Layout is designed to both horizontally and vertically center a single child element within its parent.
```

Comment:
You can remove the split infinitive something like this - "The Bullseye Layout is designed to center a single child element both horizontally and vertically within its parent."